phase3/weave-net/weave-net-list.json

```diff
@@ -0,0 +1,351 @@
+                },
+                "namespace": "kube-system"
+            }
+        },
+        {
+            "apiVersion": "rbac.authorization.k8s.io/v1beta1",
+            "kind": "ClusterRole",
+            "metadata": {
+                "name": "weave-net",
+                "annotations": {
+                    "cloud.weave.works/launcher-info": "{\n  \"original-request\": {\n    \"url\": \"/k8s/v1.8/net.yaml?v=2.4.1\",\n    \"date\": \"Tue Sep 11 2018 02:00:56 GMT+0000 (UTC)\"\n  },\n  \"email-address\": \"support@weave.works\"\n}"
+                },
+                "labels": {
+                    "name": "weave-net"
+                }
+            },
+            "rules": [
+                {
+                    "apiGroups": [
+                        ""
+                    ],
+                    "resources": [
+                        "pods",
+                        "namespaces",
+                        "nodes"
+                    ],
+                    "verbs": [
+                        "get",
+                        "list",
+                        "watch"
+                    ]
+                },
+                {
+                    "apiGroups": [
+                        "networking.k8s.io"
+                    ],
+                    "resources": [
+                        "networkpolicies"
+                    ],
+                    "verbs": [
+                        "get",
+                        "list",
+                        "watch"
+                    ]
+                },
+                {
+                    "apiGroups": [
+                        ""
+                    ],
+                    "resources": [
+                        "nodes/status"
+                    ],
+                    "verbs": [
+                        "patch",
+                        "update"
+                    ]
+                }
+            ]
+        },
+        {
+            "apiVersion": "rbac.authorization.k8s.io/v1beta1",
+            "kind": "ClusterRoleBinding",
+            "metadata": {
+                "name": "weave-net",
+                "annotations": {
+                    "cloud.weave.works/launcher-info": "{\n  \"original-request\": {\n    \"url\": \"/k8s/v1.8/net.yaml?v=2.4.1\",\n    \"date\": \"Tue Sep 11 2018 02:00:56 GMT+0000 (UTC)\"\n  },\n  \"email-address\": \"support@weave.works\"\n}"
+                },
+                "labels": {
+                    "name": "weave-net"
+                }
+            },
+            "roleRef": {
+                "kind": "ClusterRole",
+                "name": "weave-net",
+                "apiGroup": "rbac.authorization.k8s.io"
+            },
+            "subjects": [
+                {
+                    "kind": "ServiceAccount",
+                    "name": "weave-net",
+                    "namespace": "kube-system"
+                }
+            ]
+        },
+        {
+            "apiVersion": "rbac.authorization.k8s.io/v1beta1",
```

Comment:
v1beta1 ?

Reply:
i'm not sure. just downloaded and converted to JSON from YAML.